darwin/dataset/remote_dataset.py

```diff
@@ -253,18 +254,13 @@ def remove_remote(self):
         """Archives (soft-deletion) the remote dataset"""
         self.client.put(f"datasets/{self.dataset_id}/archive", payload={}, team=self.team)
 
-    def get_report(self, granularity="day"):
-        return self.client.get(
-            f"/reports/{self.dataset_id}/annotation?group_by=dataset,user&dataset_ids={self.dataset_id}&granularity={granularity}&format=csv&include=dataset.name,user.first_name,user.last_name,user.email",
-            team=self.team,
-            raw=True,
-        ).text
-
-    def release(self, name: Optional[str] = None):
+    def export(self, annotation_class_ids: Optional[List] = None, name: Optional[str] = None):
```

Comment:
Name shouldn't be optional (otherwise the backend sends: `{'errors': {'name': ["can't be blank"]}}`
I would also change `Optional[List]` to `Optional[List[str]]`
and change the order of the arguments: `export(self, name: str, annotation_class_ids: Optional[List[str]])`

Reply:
Also the name is more like a slug (it doesn't have spaces) @andreaazzini was that check only done on the frontend side? I seem to be able to create "test test" ?

Reply:
I adopted the proposed changes. Regarding the name vs slug notation, I used name because that's what's written in the payload for the post request sniffed from the front end, but it can surely be changed.